### ISSUE TITLE
Add Style Scheme Manager

### ISSUE THREAD

**Issue by ryanleesipes** [evolve-os/journal#48]
Want to add some kind of style scheme manager to allow users to install more themes as they please.

**Comment by zesterer** [comment]
Probably best to let it keep using Gtk's style scheme manager for inter-app consistency purposes.

**Comment by ryanleesipes** [comment]
I agree. I think I'm going to create something that lets people download
new ones.

**Comment by zesterer** [comment]
Ok, just so long as it runs through Gtk (and hence applies to every other program)
